### PR TITLE
fix: popups ui

### PR DESCRIPTION
**Pull request by nuanyang233**
## Description

Closes #MASK-443 MASK-448 MASK-465 MASK585
 
## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
  - [x] I have removed all in development `console.log`s
  - [x] I have removed all commented code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/plugins/Wallet/SNSAdaptor/WalletStatusDialog/TransactionDescription.tsx [hunk 1]
```diff
@@ -72,6 +73,8 @@ function getTransactionDescription(
                     )
                     const amountOut = formatBalance(computedPayload.parameters.amountOutMin, tokenDetailed?.decimals, 2)
                     return `Swap ${amountIn} ${nativeTokenDetailed?.symbol} for ${amountOut} ${tokenDetailed?.symbol}`
+                case 'create_red_packet':
+                    return 'Lucky drop'
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Wallet/SNSAdaptor/WalletStatusDialog/TransactionDescription.tsx]: `Create lucky drop with ${amount} ${symbol}.`

File: packages/mask/src/plugins/Wallet/SNSAdaptor/WalletStatusDialog/TransactionDescription.tsx [hunk 2]
```diff
@@ -74,7 +74,11 @@ function getTransactionDescription(
                     const amountOut = formatBalance(computedPayload.parameters.amountOutMin, tokenDetailed?.decimals, 2)
                     return `Swap ${amountIn} ${nativeTokenDetailed?.symbol} for ${amountOut} ${tokenDetailed?.symbol}`
                 case 'create_red_packet':
-                    return 'Lucky drop'
+                    return `Create lucky drop with ${formatBalance(
```
Comment (septs) [inline packages/mask/src/plugins/Wallet/SNSAdaptor/WalletStatusDialog/TransactionDescription.tsx]: plz extract formatted balance as variable

File: packages/mask/src/extension/popups/pages/Wallet/GasSetting/Prior1559GasSetting.tsx [hunk 3]
```diff
@@ -87,6 +89,11 @@ const useStyles = makeStyles()((theme) => ({
     },
 }))
 
+const minGasPriceOfChain: { [key in ChainId]?: string } = {
```
Comment (septs) [inline packages/mask/src/extension/popups/pages/Wallet/GasSetting/Prior1559GasSetting.tsx]: extract `{ [key in ChainId]?: string }` as shared type?

e.g:
```ts
type ChainIdRecord<T> = { [key in ChainId]?: T } 
```

File: packages/mask/src/extension/popups/components/StyledInput/index.tsx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { forwardRef, memo } from 'react'
 import { TextFieldProps, TextField } from '@mui/material'
 import { makeStyles, useStylesExtends } from '@masknet/theme'
+import classnames from 'classnames'
```
Comment (septs) [inline packages/mask/src/extension/popups/components/StyledInput/index.tsx]:
```suggestion
import classNames from 'classnames'
```

File: packages/mask/src/extension/popups/pages/Wallet/Transfer/Prior1559Transfer.tsx [hunk 5]
```diff
@@ -144,13 +153,14 @@ export const Prior1559Transfer = memo<Prior1559TransferProps>(({ selectedAsset,
...
+    const [addressTip, setAddressTip] = useState<{ type: TransferAddressError; message: string } | null>()
+
     const schema = useMemo(() => {
         return zod.object({
             address: zod
                 .string()
                 .min(1, t('wallet_transfer_error_address_absence'))
-                .refine((address) => EthereumAddress.isValid(address), t('wallet_transfer_error_invalid_address'))
-                .refine((address) => address !== wallet?.address, t('wallet_transfer_error_address_absence')),
+                .refine((address) => EthereumAddress.isValid(address), t('wallet_transfer_error_invalid_address')),
```
Comment (septs) [inline packages/mask/src/extension/popups/pages/Wallet/Transfer/Prior1559Transfer.tsx]:
```suggestion
                .refine(EthereumAddress.isValid, t('wallet_transfer_error_invalid_address')),
```

File: packages/mask/src/extension/popups/pages/Wallet/Transfer/Prior1559Transfer.tsx [hunk 6]
```diff
@@ -272,6 +313,24 @@ export const Prior1559Transfer = memo<Prior1559TransferProps>(({ selectedAsset,
...
+                                ? classes.normalMessage
+                                : classes.errorMessage
+                        }>
+                        {addressTip.message}
+                    </Typography>
+                </Box>
+            )
+        }
+
+        return
```
Comment (septs) [inline packages/mask/src/extension/popups/pages/Wallet/Transfer/Prior1559Transfer.tsx]:
```suggestion
        if (!addressTip) return
            return (
                <Box py={2.5} px={1.5}>
                    <Typography
                        className={
                            addressTip.type === TransferAddressError.SAME_ACCOUNT
                                ? classes.normalMessage
                                : classes.errorMessage
                        }>
                        {addressTip.message}
                    </Typography>
                </Box>
            )
```

File: packages/mask/src/extension/popups/pages/Wallet/Transfer/Prior1559Transfer.tsx [hunk 7]
```diff
@@ -343,8 +402,8 @@ export const Prior1559TransferUI = memo<Prior1559TransferUIProps>(
         } = useFormContext<TransferFormData>()
 
         useUpdateEffect(() => {
-            setPopoverOpen(hasEnsSuffix && !!anchorEl.current)
-        }, [hasEnsSuffix])
+            setPopoverOpen(!!popoverContent && !!anchorEl.current)
```
Comment (septs) [inline packages/mask/src/extension/popups/pages/Wallet/Transfer/Prior1559Transfer.tsx]:
```suggestion
            setPopoverOpen(Boolean(popoverContent && anchorEl.current))
```

File: packages/mask/src/extension/popups/pages/Wallet/GasSetting/Prior1559GasSetting.tsx (hunk 3 above)
Comment (nuanyang233) [reply packages/mask/src/extension/popups/pages/Wallet/GasSetting/Prior1559GasSetting.tsx:92]: No, we have many of the same types, but the `value` has a different meaning.

**Review by septs — CHANGES_REQUESTED** [review]

**Review by guanbinrui — DISMISSED** [review]

**Review by Jack-Works — DISMISSED** [review]

**Comment by wenluomask** [comment]
Checked.

File: packages/mask/src/extension/popups/pages/Wallet/components/ActivityList/ActivityListItem.tsx [hunk 8]
```diff
@@ -1,9 +1,9 @@
 import type { RecentTransaction } from '../../../../../../plugins/Wallet/services'
 import { makeStyles } from '@masknet/theme'
-import React, { memo } from 'react'
+import React, { memo, useMemo } from 'react'
```
Comment (septs) [inline packages/mask/src/extension/popups/pages/Wallet/components/ActivityList/ActivityListItem.tsx]:
```suggestion
import { memo, useMemo } from 'react'
```

**Review by Lanttcat — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Comment by orange187** [comment]
Checked

**Comment by orange187** [comment]
Checked

**Pull request merged**